app/assets/src/components/views/SampleUploadFlow/ReviewStep.jsx

```diff
@@ -131,6 +131,7 @@ class ReviewStep extends React.Component {
   render() {
     const { showUploadModal, showLessDescription } = this.state;
     const shouldTruncateDescription =
+      this.props.project.description &&
```

Comment:
nit: it'd be nice to still have the `this.props` consts pulled out so you don't have to repeat `this.props.project.description` throughout

Reply:
I think you should also modify `countNewLines` to just return 0 if it has a null argument. It can pay to be defensive in case that function is used in the future